src/client/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest AS build
+WORKDIR /usr/src/app
+COPY package*.json ./
```

Comment:
For make it for the convenience sake, lets copy package.json to the same root app root (COPY package.json /usr/src/app/package.json)